data/denis-p_j144137.2-094558.json

```diff
@@ -16,6 +16,9 @@
         {
             "other_name": "2MASS J14413716-0945590"
         },
+        {
+            "other_name": "2MASS J14413716\u22120945590"
```

Comment:
This other_name is the same as one that's already there, except that it has the unicode character for a minus sign (\u2212) instead. Not something we need to fix for this PR, but we will want to address that in the future.

Reply:
Could I just delete the other version,  I know that in theory they  are different since one has the unicode character but I think it may be a bit confusing to keep it so it may make more sense to take out the other name with the unicode